Resources/config/security.xml

```diff
@@ -6,6 +6,7 @@
 
     <parameters>
         <parameter key="fos_oauth_server.security.authentication.provider.class">FOS\OAuthServerBundle\Security\Authentication\Provider\OAuthProvider</parameter>
+        <parameter key="fos_oauth_server.security.authentication.listener.legacy.class">FOS\OAuthServerBundle\Security\Firewall\LegacyOAuthListener</parameter>
```

Comment:
Ok I'll put it directly on Symfony definition. I don't know that is discouraged also for external bundles.